File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.java
class Solution {
    public int longestConsecutive(int[] nums) {
        int n = nums.length;
        if(n == 0) return 0;
        Arrays.sort(nums);
        
        int num = nums[0];
        int i = 0; 
        int len = 0;
        int max = 1;

        while(i < n){
            if(num != nums[i]){
                num = nums[i];
                len = 0;
            }

            while(i < n && num == nums[i]){
                i++;
            }
            len++;
            num++;
            max = Math.max(len, max);
        }

        return max;
    }
}

File: Data Structures & Algorithms/move-zeroes/submission-2.cs
public class Solution {
    public void MoveZeroes(int[] nums) {
        int n = nums.Length;
        int i = 0;
        int j = 0;
        for(i = 0; i < n; i++){
            if(nums[j] == 0 && nums[i] != 0){
                nums[j] = nums[i];
                nums[i] = 0;
                j++;
            }
            if(nums[j] != 0) j++;
        }
    }
}